hasher-matcher-actioner/webapp/src/components/settings/ActionRuleFormColumns.tsx

```diff
@@ -3,14 +3,36 @@
  */
 
 import React from 'react';
-import {PropTypes} from 'prop-types';
+import PropTypes from 'prop-types';
 import Form from 'react-bootstrap/Form';
 import Row from 'react-bootstrap/Row';
 import Col from 'react-bootstrap/Col';
 import Button from 'react-bootstrap/Button';
 
 export const classificationTypeTBD = 'TBD';
 
+type Action = {
+  id: string;
+  name: string;
+};
+
+type Classification = {
+  classificationType: string;
+  equals: boolean;
```

Comment:
is it confusing what this fields is saying, equal_to vs not_equal_to check when a match is found?

Reply:
Gonna rename the class and the field